### DISCUSSION TITLE
REQ: HUD - (team)frags - fixed_order and outward text alignment

### DISCUSSION THREAD

**Discussion by AvtursIoni** [ezQuake/ezquake-source#511] [Ideas]
`(team)frags fliptext 2`
Inward frag counts (cells) alignment or outward text alignment. 2 columns only.
`(team)frags fixed_order`
Just like score_bar's fixed_order.

**Comment by JamieGirardd** [comment]
Es scheint, als ob die Idee darin bestünde, dem Element „Team-Frags“ mehr Kontrolle über das Layout zu geben. Eine Option wie „fliptext 2“, mit der die Frag-Zähler entweder nach innen zur Mitte oder nach außen zu den Rändern ausgerichtet werden können, würde HUD-Setups deutlich übersichtlicher gestalten, insbesondere bei symmetrischen Designs. Die Beschränkung auf zwei Spalten sorgt für Einfachheit und Konsistenz. Ein Parameter für die feste Reihenfolge, wie er bereits in der Scorebar vorhanden ist, würde zudem verhindern, dass die Teams dynamisch ihre Positionen tauschen. Spinanga Casino https://spinanga-de.de/ steht für zügiges Spielvergnügen und dynamische Momente. Dies ist besonders nützlich für benutzerdefinierte Overlays und Wettkampfübertragungen. Insgesamt handelt es sich um eine kleine Anpassung, die HUD-Designern jedoch mehr präzise Kontrolle ermöglicht.
